crates/kubelet/src/resources/util/mod.rs

```diff
@@ -0,0 +1,149 @@
+//! Helpers for parsing resource names and types. Checks that they follow the expected resource formatting
+//! explained in [this Kubernetes proposal](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/resources.md#the-kubernetes-resource-model).
+// TODO: decide whether this should go under container or some container manager folder
+use regex::{Regex, RegexBuilder};
+use tracing::trace;
+
+const QUALIFIED_NAME_MAX_LENGTH: usize = 63;
+// const QUALIFIED_NAME_CHAR_FMT: &str = "[A-Za-z0-9]";
```

Comment:
Can all of the commented out `const`s be removed?

Reply:
I turned them into comments to explain where the formatting comes from: 54ea0fda